docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerJobProperty.java

```diff
@@ -14,6 +15,12 @@
 public class DockerJobProperty extends JobProperty<AbstractProject<?, ?>> {
 
     /**
+     * Image repository name for committed containers (image names)
+     */
+    @DataBoundSetter
+    public String imageRepository;
```

Comment:
You should follow docs, jelly checks for public field and then for getter/setter AFAIR. But again, using public fields is bad practise.